File: templates/php/lib/HttpClient/Response.php
<?php

namespace {{call .Fnc.camelize .Pkg.Name}}\HttpClient;

/*
 * Response object contains the response returned by the client
 */
class Response
{
    function __construct($body, $code, $headers) {
        $this->body = $body;
        $this->code = $code;
        $this->headers = $headers;
    }
}

File: templates/ruby/lib/http_client/response.rb
module {{call .Fnc.camelize .Pkg.Name}}

  module HttpClient

    # Response object contains the response returned by the client
    class Response

      attr_accessor :body, :code, :headers

      def initialize(body, code, headers)
        @body = body
        @code = code
        @headers = headers
      end

    end

  end

end
